### DISCUSSION TITLE
Punchy 0.2.0 Release Planning

### DISCUSSION THREAD

**Discussion by odecay** [fishfight/punchy#263] [Planning]
Planning for [0.2.0 Release](https://github.com/fishfolks/punchy/milestone/8).

If anyone wants to post what they are working on or needs help finding something to work on in this release window feel free to discuss here.

**Comment by odecay** [comment]
I'm going to start working on #217 Networking with https://github.com/lucaspoffo/renet, but I will almost certainly not have it ready for 0.2.0.
I may also take some time away from that to work on #179.

**Comment by Zac8668** [comment]
I'm gonna start with #180, after that maybe #89, I think it would add a lot to the game, players could choose a fighter and maybe add the hats to the join menu, more customization, more fun!
